includes/admin/reporting/export/class-batch-export-earnings-report.php

```diff
@@ -212,7 +212,7 @@ public function print_csv_rows() {
 			}
 
 			// Allows extensions with other 'completed' statuses to alter net earnings, like recurring.
-			$completed_statuses = apply_filters( 'edd_export_earnings_completed_statuses', array( 'complete', 'revoked' ) );
+			$completed_statuses = apply_filters( 'edd_export_earnings_completed_statuses', edd_complete_order_statuses() );
```

Comment:
`edd_complete_order_statuses` is returning a different kind of array than what you're replacing.

Reply:
@robincornett Right this is one of those things I updated, I need to change it to the keys.